samwise.js

```diff
@@ -1,19 +1,20 @@
 /**
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.